source/opt/passes.h

```diff
@@ -74,6 +79,132 @@ class EliminateDeadConstantPass : public Pass {
   bool Process(ir::Module*) override;
 };
 
+// The optimization pass for folding SpecConstantOp and SpecConstantComposite
+// instructions. After freezing, spec constants defined with Spec ID are
```

Comment:
I feel we should not mention freezing spec constants here since that will make this pass sound depending on freeze-spec-const-pass. (Although in fact it's beneficial to do that first.) What about delete the sentence starting with "after freezing" and change the next one as "... whose elements are all known normal constants"?